cpp/arcticdb/version/local_versioned_engine.cpp

```diff
@@ -895,11 +897,11 @@ folly::Future<folly::Unit> LocalVersionedEngine::delete_trees_responsibly(
 
     storage::ReadKeyOpts read_opts;
     read_opts.ignores_missing_key_ = true;
-    auto data_keys_to_be_deleted = get_data_keys_set(store(), *keys_to_delete, read_opts);
+    auto data_keys_to_be_deleted = get_data_keys_set(store, *keys_to_delete, read_opts);
```

Comment:
I've renamed this function in #1657, whoever merges second will need to rebase

Reply:
I'll rebase it once the PR is approved